independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanDeployment.java

```diff
@@ -203,8 +204,12 @@ public ContextConfigurator configure(Class<? extends Annotation> scopeAnnotation
 
     BeanRegistrar.RegistrationContext registerBeans(List<BeanRegistrar> beanRegistrars) {
         List<InjectionPointInfo> injectionPoints = new ArrayList<>();
-        this.beans.addAll(findBeans(initBeanDefiningAnnotations(beanDefiningAnnotations, stereotypes.keySet()), observers,
-                injectionPoints, jtaCapabilities));
+        final List<BeanInfo> addedBeans = findBeans(initBeanDefiningAnnotations(beanDefiningAnnotations, stereotypes.keySet()),
```

Comment:
What's the reason for this?

Reply:
It is a leftover after some debug output that was there to check whether the order stays stable during repeated builds. Let me revert it.

Reply:
Reverted in f8b3805247e0c5b24e1f2d1ef29fdfdf7046a09e